SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# CORTX Security Policy
+
+We take the security of the CORTX project seriously. If you find any security vulnerabilities in any of the CORTX-owned source code managed via our [GitHub Organisation](https://github.com/Seagate) repositories, please report it to us as described below.
```

Comment:
Remove the word "Organisation" from "our GitHub Organisation repositories". It's confusing.

Reply:
@andriytk FYI: "[organisation](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/about-organizations)" is conventional GitHub terminology for non-individual accounts